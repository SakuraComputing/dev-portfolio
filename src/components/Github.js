import React from 'react';

export const Github = () => (
    <section className="github section">
        <div className="section-inner">
            <h2 className="heading">My GitHub</h2>
            <p>You can embed your GitHub contribution graph calendar using IonicaBizau's <a
                href="https://github.com/IonicaBizau/github-calendar" target="_blank">GitHub Calendar</a> widget.

                <div id="github-graph" className="github-graph">

                </div>

                <p>You can also embed your GitHub activities using Casey Scarborough's <a
                    href="http://caseyscarborough.com/projects/github-activity/" target="_blank">GitHub Activity
                    Stream</a> widget.
                </p>
                <div id="ghfeed" className="ghfeed">

                </div>
            </p>
        </div>
    </section>
);

export default Github;