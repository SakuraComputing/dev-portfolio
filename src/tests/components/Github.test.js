import React from 'react';
import { shallow } from 'enzyme';
import Github from '../../components/Github';

describe('Github Section', () => {
    it('should render correctly', () => {
        const wrapper = shallow(<Github />);
        expect(wrapper).toMatchSnapshot();
    });
});