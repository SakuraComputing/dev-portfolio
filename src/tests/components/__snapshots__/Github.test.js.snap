// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Github Section should render correctly 1`] = `
<section
  className="github section"
>
  <div
    className="section-inner"
  >
    <h2
      className="heading"
    >
      My GitHub
    </h2>
    <p>
      You can embed your GitHub contribution graph calendar using IonicaBizau's 
      <a
        href="https://github.com/IonicaBizau/github-calendar"
        target="_blank"
      >
        GitHub Calendar
      </a>
       widget.
      <div
        className="github-graph"
        id="github-graph"
      />
      <p>
        You can also embed your GitHub activities using Casey Scarborough's 
        <a
          href="http://caseyscarborough.com/projects/github-activity/"
          target="_blank"
        >
          GitHub Activity Stream
        </a>
         widget.
      </p>
      <div
        className="ghfeed"
        id="ghfeed"
      />
    </p>
  </div>
</section>
`;
